workloads/kube-burner-ocp-wrapper/run.sh

```diff
@@ -122,5 +122,5 @@ if [ $exit_code -eq 0 ]; then
 else
   JOB_STATUS="failure"
 fi
-env JOB_START="$JOB_START" JOB_END="$JOB_END" JOB_STATUS="$JOB_STATUS" UUID="$UUID" WORKLOAD="$WORKLOAD" ES_SERVER="$ES_SERVER" ../../utils/index.sh
```

Comment:
Please do not remove these `EVN`s, this will effect PROW runs. We need to look at a possibility, if the same set of `ENV`s be used by jenkins runs as well. It should be possible

Reply:
I set the "set - a" option at the top of this file, I believe that should set all variables to environment variables so we dont need to double specify them. I was able to see these variables in jenkins with no issues so I would think prow would be no different

Reply:
No setting `set -a` will export all the variables which might overwrite some of the already exported once (with same name) as a part of CI pipeline (involving multiple steps) which might results into errors. So reduce those changes we are sending those ENVs explicitly.